### DISCUSSION TITLE
感觉 5 月份能全部完稿

### DISCUSSION THREAD

**Discussion by isno** [isno/theByteBook#163] [General]
之前定了个 2 月份完稿，过于乐观了。

-  ServiceMesh 2月份可以写完
- 可观测，加把劲 3月3号之前写完
-  容器这一部分，用一个月时间
-  Gitops 用半个月

时间已经来到 4月中旬了。再用一个半月时间梳理，终稿差不多在 6月之前。

整部书完整用了 2年时间，其实算不上长。

**Comment by hexody** [comment]
作者是全职在写吗，还是业余时间写的

**Reply by isno** [reply]
我业余时间写

**Comment by yshdzw** [comment]
7.4 节可以补充一下不？

容器本质上就是一个特殊的进程，通过 namespace 实现资源（PID、User、网络、文件系统等）隔离，通过 cgroups 实现资源（CPU、内存、磁盘、网络带宽）限制。通过 OverlayFS 实现将只读的基础文件系统（镜像）和可写的容器文件系统叠加合并，形成轻量化、高效管理的联合文件系统。

**Reply by isno** [reply]
容器这一章会在3月写完，3月份之后再重新看容器内容。

**Comment by yshdzw** [comment]
对了， ingress 都写了，gateway api 也可以写一下 ：）

**Reply by isno** [reply]
你看的是历史内容吧，ingress 我都删除了

**Comment by Snowmoons** [comment]
好看 爱看 对想转云方向的传统运维很有帮助
